### PR TITLE
Coverity: Prevent NULL dereferences and other stuffs. 

### PR DESCRIPTION
**Pull request by ssakash**
***Fixes the following coverity Issues :-***

- CID 146839 (#1 of 1): Explicit null dereferenced (FORWARD_NULL)11. var_deref_model: Passing null pointer fb_pages to UsePages, which dereferences it.

- CID 146840 (#1 of 1): Explicit null dereferenced (FORWARD_NULL)11. var_deref_model: Passing null pointer zb_pages to UsePages, which dereferences it.

- CID 146834 (#2-1 of 2): Division or modulo by zero (DIVIDE_BY_ZERO)9. divide_by_zero: In expression tpf * 10000ULL / ttpf, division by expression ttpf which may be zero has undefined behavior.

- CID 146889 (#1 of 1): Dereference null return value (NULL_RETURNS)4. dereference: Dereferencing a pointer that might be null wxGetApp()->GetDisassemblyPtr() when calling update

### CODE REVIEW

File: pcsx2/DebugTools/Breakpoints.cpp [hunk 1]
```diff
@@ -392,6 +392,7 @@ void CBreakPoints::Update(u32 addr)
 	
 	if (resume)
 		r5900Debug.resumeCpu();
-
-	wxGetApp().GetDisassemblyPtr()->update();
+	auto disassembly_window = wxGetApp().GetDisassemblyPtr();
+	if (disassembly_window) // make sure that valid pointer is recieved to prevent potential NULL dereference.
+		wxGetApp().GetDisassemblyPtr()->update();
```
Comment (gregory38) [inline pcsx2/DebugTools/Breakpoints.cpp]: Just reuse the temp variable.

Comment (ssakash) [inline pcsx2/DebugTools/Breakpoints.cpp]: I'm getting dumb -_- , nice catch @gregory38  

should be fixed :+1:

File: plugins/GSdx/GSFunctionMap.h [hunk 2]
```diff
@@ -125,7 +125,7 @@ template<class KEY, class VALUE> class GSFunctionMap
 			KEY key = i->first;
 			ActivePtr* p = i->second;
 
-			if(p->frames > 0)
+			if( p->frames && ttpf )
```
Comment (gregory38) [inline plugins/GSdx/GSFunctionMap.h]: remove space after ```(``` and before ```)```

Comment (avih) [inline plugins/GSdx/GSFunctionMap.h]: Also add space between the `if` and the `(`.

Comment (ssakash) [inline plugins/GSdx/GSFunctionMap.h]: @avih : other ```if`()`` checks at the code aren't using that format.

Comment (gregory38) [inline plugins/GSdx/GSFunctionMap.h]: Yes GSdx doesn't use this convention. That why I didn't ask you to change it. I know ref change a couple of them. And personally I might have change some others too (in particular in my code ;)).

Comment (ssakash) [inline plugins/GSdx/GSFunctionMap.h]: Fixed

**Comment by gregory38** [comment]
Ok good. Thanks.

**Pull request merged**